ctapipe/image/muon/muon_ring_finder.py

```diff
@@ -69,3 +180,16 @@ def fit(self, x: u.deg, y: u.deg, weight, times=None):
         output.ring_fit_method = "ChaudhuriKundu"
 
         return output
+
+    def fit(self, x, y, img):
```

Comment:
this should be `__call__`. It  should not not save `x` and `y` into the component. That state tracking is not necessary.

Reply:
thx, done.